gnome-initial-setup/pages/keyboard/cc-input-chooser.c

```diff
@@ -404,7 +408,7 @@ get_locale_infos (CcInputChooser *chooser)
 	GList *list;
 
 	if (gnome_get_input_source_from_locale (priv->locale, &type, &id)) {
-                add_row_to_list (chooser, type, id);
+	  add_row_to_list (chooser, type, id, FALSE);
```

Comment:
Indentation got messed up here too. I hate tabs. :( Be careful to ensure tabs are set to 8 spaces in your editor.